mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -136,6 +138,30 @@ def parse_json_input(json_input, orient="split", schema: Schema = None):
         )
 
 
+def parse_jsonlines_input(jsonlines_input, schema: Schema = None):
+    """
+    :param jsonlines_input: A line-delimited JSON-formatted string representation of
+                      a Pandas DataFrame, or a stream containing such a string representation.
+    :param schema: Optional schema specification to be used during parsing.
+    """
+
+    try:
+        if schema is None:
+            return pd.read_json(jsonlines_input, dtype=False, precise_float=False, lines=True)
+        else:
+            dtypes = dict(zip(schema.input_names(), schema.pandas_types()))
+            return pd.read_json(jsonlines_input, dtype=dtypes, precise_float=False, lines=True)
```

Comment:
Looks like we're missing support for Tensor jsonlines inputs, which is implemented in `_dataframe_from_json` (https://github.com/mlflow/mlflow/blob/e1d26161858e3af3114c36e6d346840753a7157e/mlflow/utils/proto_json_utils.py#L196).

Instead of defining a new `parse_jsonlines_input` method, can we add a `lines=True/False` argument to `parse_json_input` and `_dataframe_from_json`?